client/src/stores/historyStore.ts

```diff
@@ -149,7 +149,8 @@ export const useHistoryStore = defineStore(
             const deletedHistory = (await deleteHistoryById(historyId, purge)) as HistorySummary;
             Vue.delete(storedHistories.value, deletedHistory.id);
             if (getFirstHistoryId.value) {
-                return setCurrentHistoryId(getFirstHistoryId.value);
+                setCurrentHistoryId(getFirstHistoryId.value);
+                return setCurrentHistory(getFirstHistoryId.value);
```

Comment:
This seems strange, as `setCurrentHistory` also calls `setCurrentHistoryId` via `selectHistory`, so it is now being called twice.

What is the reason for this change, and could it also be done within `setCurrentHistory` or `selectHistory`?